test/cmd/create.sh

```diff
@@ -153,3 +153,63 @@ run_kubectl_create_kustomization_directory_tests() {
   set +o nounset
   set +o errexit
 }
+
+# Runs tests related to kubectl create --validate
+run_kubectl_create_validate_tests() {
```

Comment:
You might need to add this function in here https://github.com/kubernetes/kubernetes/blob/master/test/cmd/legacy-script.sh to be run.

Reply:
thanks, done